rpc/client/rpc_test.go

```diff
@@ -538,30 +538,52 @@ func TestBlockSearch(t *testing.T) {
 
 func TestTxStatus(t *testing.T) {
```

Comment:
[optional] should this include an assertion or test case for evicted transactions?

Reply:
yes should be good!

Reply:
so, the mempool there uses default mempool config where the cache size is 5000 txs. I couldn't overflow that in this unit test and also didn't see a way of changing the mempool. 

any suggestions welcome